src/interface/viewport.c

```diff
@@ -1703,9 +1694,9 @@ void viewport_banner_paint_setup(uint8 direction, int height, rct_map_element* m
 	direction += map_element->properties.banner.position;
 	direction &= 3;
 
-	RCT2_GLOBAL(0x9DEA56, uint16_t) = height + 2;
-
-	RCT2_GLOBAL(0x9DEA52, uint32_t) = RCT2_ADDRESS(0x98D884, uint32)[direction * 2];
+	boundBoxOffsetX = RCT2_ADDRESS(0x98D884, uint16)[direction * 2];
```

Comment:
direction * 4 as we are now referencing a uint16 instead of uint32. This was the source of #3345.